simple-data-backend/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2022,2024: Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+# Copyright (c) 2021,2024 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0. *
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# * SPDX-License-Identifier: Apache-2.0
+
```

Comment:
The image doesn't build successfully, see https://github.com/eclipse-tractusx/tractus-x-umbrella/actions/runs/8378208161/job/22942230800#step:3:232

Reply:
thanks, I adjusted that and did some additional changes to the test workflow for get it to run with the container registry, still the install of the chart fails https://github.com/eclipse-tractusx/tractus-x-umbrella/actions/runs/8391972593/job/22983537342#step:10:245
it would be very good to have the chart in a state where it can be tested via workflow, otherwise it's hard to maintain.

Reply:
I changed the URL in the helm test hook to "/actuator/health". This way the it will be checked, if the application could be started